tests/index.js

```diff
@@ -31,6 +31,7 @@ require('./testShowClosedAssessmentScore');
 require('./testActiveAccessRestriction');
 require('./testGradeRate');
 require('./testBonusPoints');
+require('./testScrapPaper'); // must go before testCron.js or breakage only in CI/CD pipeline; view commits before and after #541987f
 require('./testAccess');
```

Comment:
For some reason, if the testCron.js job runs first, the after() function in the tests cannot complete in `testScrapPaper.js`.

Reply:
This only occurs in CI/CD. Inside the docker container on my laptop, this is not an issue.